package-lock.json

```diff
@@ -4960,6 +4960,18 @@
         "@octokit/openapi-types": "^18.0.0"
       }
     },
+    "node_modules/@phosphor-icons/react": {
+      "version": "2.1.7",
+      "resolved": "https://registry.npmjs.org/@phosphor-icons/react/-/react-2.1.7.tgz",
+      "integrity": "sha512-g2e2eVAn1XG2a+LI09QU3IORLhnFNAFkNbo2iwbX6NOKSLOwvEMmTa7CgOzEbgNWR47z8i8kwjdvYZ5fkGx1mQ==",
+      "engines": {
+        "node": ">=10"
+      },
+      "peerDependencies": {
+        "react": ">= 16.8",
+        "react-dom": ">= 16.8"
+      }
+    },
```

Comment:
nice! just one question: do we need to update it here?

Reply:
This dependency was just updated with the `npm install` and moved in the file (automatically)